fbu-app/PostSearch/ViewControllers/BillDetailsViewController.m

```diff
@@ -0,0 +1,148 @@
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath{
+    
+    DebtorCell *cell = [tableView dequeueReusableCellWithIdentifier:@"DebtorCell"];
+    
+    Persona *housemate = self.people[indexPath.row];
+    [housemate fetchIfNeededInBackgroundWithBlock:^(PFObject * _Nullable object, NSError * _Nullable error) {
+        
+        NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
+        [numberFormatter setNumberStyle: NSNumberFormatterCurrencyStyle];
+        
+        cell.nameLabel.text = [self getName:housemate];
+        
+        PFFileObject *imageFile = housemate.profileImage;
+        [imageFile getDataInBackgroundWithBlock:^(NSData * _Nullable data, NSError * _Nullable error) {
+            if (!error) {
+                cell.pictureView.image = [UIImage imageWithData:data];
+            }
+        }];
+        cell.pictureView.layer.cornerRadius = cell.pictureView.frame.size.height /2;
+        cell.pictureView.layer.masksToBounds = YES;
+        
+        if(indexPath.row == 0){
+            
+            [cell.extraTextLabel setHidden:NO];
+            [cell.extraMoneyLabel setHidden:NO];
+            
+            cell.owesLabel.text = @" paid ";
+            cell.portionLabel.text = [numberFormatter stringFromNumber:self.bill.paid];
+            
+            PFFileObject *imageFile = housemate.profileImage;
+            [imageFile getDataInBackgroundWithBlock:^(NSData * _Nullable data, NSError * _Nullable error) {
+                if (!error) {
+                    cell.pictureView.image = [UIImage imageWithData:data];
+                }
+            }];
+            cell.pictureView.layer.cornerRadius = cell.pictureView.frame.size.height /2;
+            cell.pictureView.layer.masksToBounds = YES;
+            
+            NSDecimalNumber *paid = [NSDecimalNumber decimalNumberWithDecimal:[self.bill.paid decimalValue]];
+            cell.extraMoneyLabel.text = [numberFormatter stringFromNumber:[paid decimalNumberBySubtracting:self.sumDebt]];
+            
+        }else{
+        
+            cell.portionLabel.text = [numberFormatter stringFromNumber:self.bill.portions[indexPath.row - 1]];
+            cell.owesLabel.text = @" owed ";
+            
+            [cell.extraTextLabel setHidden:YES];
+            [cell.extraMoneyLabel setHidden:YES];
+        }
+        
+        
+        
+    }];
+    
+    return cell;
+    
+}
+                      
+- (NSString *)formatDate:(NSDate *)date
+    {
+        NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+        [dateFormatter setDateStyle:NSDateFormatterShortStyle];
+        [dateFormatter setDateFormat:@"MMMM dd, yyyy"];
+        NSString *formattedDate = [dateFormatter stringFromDate:date];
+        return formattedDate;
+    }
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.people.count;
```

Comment:
```suggestion
    return people.count;
```

Reply:
switched them to instance variables, thanks!